tests/testthat/test-workflow-nmbayes.R

```diff
@@ -82,18 +82,16 @@ test_that("nmbayes: submit_model() checks for existing model", {
 })
 
 test_that("nmbayes: run_log() captures runs correctly", {
-  skip("FIXME: submodels confuse run_log()")
-  log_df <- run_log(".")
+  log_df <- run_log(".", .recurse = TRUE)
```

Comment:
Is the point of adding `.recurse = TRUE` here just to make sure that the nested models no longer show up? I'm assuming yes, but correct me if I'm missing something.

Reply:
Using `.recurse = TRUE` does demonstrate that the nested models don't show up, but that's not the main reason for it.  This `run_log` call uses "." as the directory, and that looks like this:

```
extdata/
model/nonmem/bayes/
```

So we need the `.recurse = TRUE` for this to get any models at all.

Here's my explanation for "how did that work before?" in the message of 4b37b8d:

> [...] update the run_log() call to pass `.recurse = TRUE` to account
> for the change in default (bbr's afa6bff6).

That lines up date-wise:

 * [ed33da2](https://github.com/metrumresearchgroup/bbr.bayes/commit/ed33da2415df1356b6ddd2101ec67c498e46cc8c) (tests: record nmbayes failures for *_log functions, 2022-11-16)

 * [afa6bff6](https://github.com/metrumresearchgroup/bbr/commit/afa6bff65f8511125c01f5176c1d60e630b2a57b) (refactor: changing all _log() functions to default to .recurse = FALSE [...], 2022-11-28)